packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -259,28 +259,35 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
       onCreateLocalStreamView && onCreateLocalStreamView(localVideoViewOptions);
     }
     return (
-      <VideoTile
-        key={localParticipant.userId}
-        userId={localParticipant.userId}
-        renderElement={
-          localVideoStream?.renderElement ? (
-            <>
-              {
-                /* @conditional-compile-remove(call-with-chat-composite) @conditional-compile-remove(local-camera-switcher) */
-                localCameraCycleButton(localVideoCameraCycleButtonProps)
-              }
-              <StreamMedia videoStreamElement={localVideoStream.renderElement} />
-            </>
-          ) : undefined
-        }
-        showLabel={!(shouldFloatLocalVideo && isNarrow)}
-        displayName={isNarrow ? '' : strings.localVideoLabel}
-        initialsName={localParticipant.displayName}
-        styles={localVideoTileStylesThemed}
-        onRenderPlaceholder={onRenderAvatar}
-        isMuted={localParticipant.isMuted}
-        showMuteIndicator={showMuteIndicator}
-      />
+      <Stack
+        tabIndex={0}
+        aria-label={'Tile keyboard movement Dropdown Hidden'}
```

Comment:
Similar to comment on aria-role, this component is more than just the Dropdown. A screen reader hitting this should convey this is a movable tile that is showing the user their curent video&mic state, not just that there's a hidden dropdown to move it around.

Reply:
(unless a11y info from the video tile child is accessible by the narrator somehow?)

Reply:
Changed this label for now to: 'movable local video tile' happy to change it again if needed! 😊